ci/tests/Jenkinsfile.e2e-nightly

```diff
@@ -49,7 +49,7 @@ pipeline {
             sh """
               python3 -m pytest -m testrail_id \
                 -m "not upgrade" \
-                -n24 --rerun_count=2 \
+                -n16 --rerun_count=2 \
```

Comment:
Parameters like `-n` mean nothing. How about adding an env variable in `environment` called `PARALLEL_SESSIONS` with the number set and then using it in the command. By using a sensible variable name we can make it clear what the flag does.

Also, maybe align the flag formatting for be the same in all 3 files for easier editing/reading.

Reply:
I just notice there isn't one here. We can add it like here:
https://github.com/status-im/status-react/blob/e7807202c93d5a1c8db28ec7846aa761e7a9a283/ci/Jenkinsfile.android#L28-L39

Reply:
So I guess after `parameters`:
```groovy
  environment {
    PARALLEL_SESSIONS = 16
  }
```
Unless you have a better name for the variable. Maybe `TEST_PARALLEL_SESSIONS` for more clarity?
Is this parameter specific to SauceLabs or any tests?

Reply:
-n is short for --numprocesses so I updated that key to make is more meaningful. It shows number of max processes to start for combines test run. I also would keep it separate from Jenkinsfile.android because env vars in Jenkinsfile.android describe variables applied for build, while -n (--numprocesses) is more related to test framework settings. It would be better in that case to specify it in pytest.ini. I keep as it is now.